build_elm_env.sh

```diff
@@ -5,36 +5,41 @@ export ELM_BUILD_DIR=`pwd -P`
 # using build_earthio_env.sh allows the
 # test data to be downloaded as well.
 build_elm_env(){
-    rm -rf .earthio_tmp;
-    git clone http://github.com/ContinuumIO/earthio .earthio_tmp && cd .earthio_tmp || return 1;
-    if [ "$EARTHIO_VERSION" = "" ];then
+    set -e
+    rm -rf .earthio_tmp
+    git clone http://github.com/ContinuumIO/earthio .earthio_tmp && cd .earthio_tmp
+    # Temporary fix
+    sed -i.bak 's/PYTHON_TEST_VERSION/PYTHON/g' ./build_earthio_env.sh
+    if [ "x$EARTHIO_VERSION" = "x" ]; then
         export EARTHIO_VERSION="master";
     fi
-    git fetch --all || return 1;
+    git fetch --all
     echo git checkout $EARTHIO_VERSION
-    git checkout $EARTHIO_VERSION || return 1;
-    conda config --set always_yes true;
-    . build_earthio_env.sh && source activate $EARTHIO_TEST_ENV || return 1;
-    conda config --set always_yes true;
-    cd $ELM_BUILD_DIR || return 1;
+    git checkout $EARTHIO_VERSION
+    set +e
+    export PYTHON=${PYTHON:-3.5}
+    export NUMPY=${NUMPY:-1.11}
+    IGNORE_ELM_DATA_DOWNLOAD=1 . build_earthio_env.sh && source activate $EARTHIO_TEST_ENV
+    set -e
+    cd $ELM_BUILD_DIR
     # End of earthio and test data related section
-    if [ "$PYTHON_TEST_VERSION" = "" ];then
-        echo FAIL - Must define PYTHON_TEST_VERSION environment variable such as 2.7, 3.5 or 3.6 - FAIL
-        return 1;
+    if [ "x$PYTHON" = "x" ]; then
```

Comment:
It distinguishes between a variable being unset versus a variable being set to the empty string. Maybe it was overkill to include here, but it's effectively checking whether the variable has both been set and is empty.